Project.toml

```
@@ -1,6 +1,6 @@
 name = "CloudWatchLogs"
 uuid = "c4c1e6a2-6bdd-52e0-b56d-1d4734724d2d"
-version = "1.1.3"
+version = "1.1.4"
```

```suggestion
version = "1.2.0"
```
Since we're now dropping a version of Memento